.github/workflows/tests.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+on:
```

Comment:
could you please simplify as in here https://github.com/rvm/rvm/blob/master/.github/workflows/specs.yml#L2

Reply:
by dropping the specified branches, we will run GitHub Actions on any temporary feature branches pushed up, even though they aren't associated with a pull request.  But I see that being rare in the fork-pull-request model since most branches won't appear directly on this repository.  I'll remove these branch references.

Reply:
Removed branches, but kept this as a multi-line so that a schedule can be defined.  Let me know if that's reasonable.

Reply:
Make sense, thanks!